src/components/ArticlesTable/ArticlesTable.js

```diff
@@ -91,11 +91,11 @@ export class ArticlesTable extends Component {
                     >
                       <MenuItem
                         disabled
-                        primaryText={`Saved: ${moment(item.get('timestamp')).format('lll')}`}
+                        primaryText={`Saved: ${moment.utc(item.get('timestamp')).local().format('lll')}`}
```

Comment:
Thank for suggestion, but from my point of view it's not needed now − I'd move it only when we actually have more instances: https://en.wikipedia.org/wiki/You_aren't_gonna_need_it